articles/connections/social/pinterest.md

```diff
@@ -5,7 +5,7 @@ image: /media/connections/pinterest.png
 seo_alias: pinterest
 description: Learn how to add login functionality to your app with Pinterest. You will need to obtain a Client ID and Client Secret for Pinterest.
 toc: true
-public: true
+public: false
```

Comment:
Removes this connection from the list but keeps the page live. This connection no longer has a record in the dashboard or in marketplace.

Reply:
The resulting page could use an update about the status of their API or just removed/404:

https://auth0.com/docs/connections/social/pinterest